js/iframeView.js

```diff
@@ -0,0 +1,101 @@
+import Adapt from 'core/js/adapt';
+import ComponentView from 'core/js/views/componentView';
+
+class iframeView extends ComponentView {
```

Comment:
Classes always have an uppercase name please.
```suggestion
class IFrameView extends ComponentView {
```